easybuild/easyconfigs/g/gompi/gompi-system-2.27.eb

```diff
@@ -0,0 +1,21 @@
+easyblock = "Toolchain"
+
+name = 'gompi'
+version = 'system'
+versionsuffix = '-2.27' # binutils version
```

Comment:
either use `system-2.27` to match GCC version, or use `system` as version and make sure the right `GCC` is loaded, otherwise you end up with a double `-2.27` in OpenMPI...

Reply:
I'll drop back to system for `OpenMPI` which is actually more accurate anyway. The toolchain used there will dictate the GCC version. For `gompi`, I think the `versionsuffix` is a valid inclusion.

Reply:
`system-2.27` would require a lot more logic in the easybuild since the `system` version is a trigger there